### PR TITLE
Adds a no-threat roundstart dynamic ruleset

### PR DESCRIPTION
**Pull request by mystery3525**
## About The Pull Request

Adds "Nothing Bad Yet" roundstart dynamic ruleset, with a cost of 0.1 threat level, which doesn't spawn any antagonists and blocks any other roundstart ruleset from being picked, causing all threat to go towards midrounds.

From: [This Thread](https://discord.com/channels/427337870844362753/1126036809634304020)

## Why It's Good For The Game

This will give the crew some breather time (at least in the beginning), and add some variation in rounds where there are just a whole bunch of midrounds and latejoins.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

dreamseeker_M3Wj9Zmvs8

dreamseeker_aFUtjJ1BKD

</details>

## Changelog
:cl:
add: New dynamic roundstart rule, "Nothing Bad Yet", which just donates all threat to midrounds
/:cl:

### CODE REVIEW

**Comment by itsmeow** [comment]
No, this is why we have a midround randomization curve. Threat given to roundstart rulesets is randomly generated from 0% to 100%, averaging at 50%

🔑

**Comment by itsmeow** [comment]
This also effectively disables every roundstart ruleset by having a weight of 100 and no requirements

File: code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm [hunk 1]
```diff
@@ -1,4 +1,28 @@
...
+//                                          //
+//            NOTHING BAD YET               //
+//                                          //
+//////////////////////////////////////////////
+
+/datum/dynamic_ruleset/roundstart/nothing
+	name = "Nothing Bad Yet"
+	required_candidates = 0
+	minimum_players = 0
+	weight = 100
```
Comment (EvilDragonfiend) [inline code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm]: I think you forgot to change this

**Review by EvilDragonfiend — CHANGES_REQUESTED** [review]

File: code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm (hunk 1 above)
Comment (mystery3525) [reply code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm]: my bad

**Comment by PowerfulBacon** [comment]
This already exists in the form of the extended ruleset making this apartment redundant

**Comment by itsmeow** [comment]
extended is different and admin only

**Pull request closed without merging**